pkg/security/podsecuritypolicy/sysctl/mustmatchpatterns.go

```diff
@@ -22,12 +22,31 @@ import (
 
 	"k8s.io/apimachinery/pkg/util/validation/field"
 	api "k8s.io/kubernetes/pkg/apis/core"
-	"k8s.io/kubernetes/pkg/apis/core/helper"
 )
 
+// SafeSysctlWhitelist returns the whitelist of safe sysctls and safe sysctl patterns (ending in *).
+//
+// A sysctl is called safe iff
+// - it is namespaced in the container or the pod
+// - it is isolated, i.e. has no influence on any other pod on the same node.
+func SafeSysctlWhitelist() []string {
```

Comment:
Perhaps, we could declare it as a const? In other words, why it should be a function?

Reply:
The whitelist can be generated by a function later? E.g. read the list from a configmap or different object?

Reply:
slices cannot be consts, they must be vars. I prefer keeping it like this. Dangers of shared slices that get appended to elsewhere (as this does): https://play.golang.org/p/TSWfCax7p6A